prototype2/gdgem/configs/vmm3.json

```diff
@@ -3,23 +3,27 @@
 
   "time_config" :
   {
-    "tac_slope" : 60,
+    "tac_slope" : 100,
     "bc_clock" : 20,
-    "trigger_resolution" : 3.125,
-    "target_resolution" : 0.5,
-    "acquisition_window" : 4000
+    "trigger_resolution" : 1,
+    "target_resolution" : 1,
+    "acquisition_window" : 8191
   },
 
   "srs_mappings" :
   [
     {"fecID":1, "vmmID":0, "planeID":0, "strip_offset":0},
     {"fecID":1, "vmmID":1, "planeID":0, "strip_offset":64},
-    {"fecID":1, "vmmID":6, "planeID":0, "strip_offset":128},
-    {"fecID":1, "vmmID":7, "planeID":0, "strip_offset":192},
-    {"fecID":1, "vmmID":10, "planeID":1, "strip_offset":0},
-    {"fecID":1, "vmmID":11, "planeID":1, "strip_offset":64},
+    {"fecID":1, "vmmID":2, "planeID":1, "strip_offset":0},
+    {"fecID":1, "vmmID":3, "planeID":1, "strip_offset":64},
```

Comment:
Should this file be edited?

Reply:
I think yes, and this has been modified to conform with doro's latest needs, which... I don't think we have a better criterion. We should probably keep a config file like this along with whatever test data we get, and that can be separate from the "recommended" one here. This should just be a file that is technically valid, as an example.

Reply:
But I think Doro changes this file every time she goes to the lab and reconfigures something. So maybe there should be more files with more descriptive names and perhaps dates?

Anyway, are we saving this in the hdf5 files? Might be a good idea.

Reply:
Yes, there is already a ticket for saving this with the h5. I think henceforth Doro will probably modify the one in essdaq more often.